sig-platform-dev/charter.md

```diff
@@ -0,0 +1,375 @@
+  *SHOULD* be supported by a majority of SIG Members.
+- Chairs *MUST* remain active in the role and are automatically removed from the position if they are
+  unresponsive for > 3 months and *MAY* be removed if not proactively working with other chairs to fulfill
+  responsibilities.
+- Number: 2+
+  - There is no upper bound on this number because this role is purely
+    one of service to the SIG, not influence over it.
+- Defined in [sigs.yaml].
+
+### SIG Technical Leads
+
+- Resolve cross-subproject technical issues and decisions.
+  - Issues within a subproject should be resolved by that subproject's owners, not the SIG TLs.
+  - Issues that require cross-subproject agreement should be resolved first by lazy consensus
+    or majority vote among subproject owners, but may be escalated to SIG TLs if agreement can't be reached.
+    - Examples of cross-subproject issues:
+      - Agreement on a cohesive vision for the PDK.
+      - Agreement on interfaces and integrations between subprojects, or on common building blocks.
+      - Changes requested in one or multiple subprojects to better align them to each other or consolidate code/effort.
+    - If subproject owners or other SIG members are not satisfied with the resolution,
+      they should escalate to SIG Architecture or the steering committee as appropriate.
+- Seed members established at SIG founding.
+- TLs *MAY* decide to step down at anytime and propose a replacement.
+  Use lazy consensus amongst subproject owners with fallback on majority vote to accept proposal.
+  This *SHOULD* be supported by a majority of SIG Members.
+- TLs *MAY* select additional TLs through a [super-majority] vote amongst TLs.
+  This *SHOULD* be supported by a majority of SIG Members.
+- TLs *MUST* remain active in the role and are automatically removed from the position if they are
+  unresponsive for > 3 months and *MAY* be removed if not proactively working with other TLs to fulfill
+  responsibilities.
+- Number: 3-5
+  - This number is bounded to ensure efficient resolution of escalations within the SIG.
+  - There must never be more than 2 TLs from the same company.
+- Defined in [sigs.yaml].
+
+### Subproject Owners
+
+- Scoped to a subproject defined in [sigs.yaml].
+- Resolve issues within their subproject when consensus cannot be reached among subproject members.
+- Collectively decide on cross-subproject issues by lazy consensus with fallback on majority vote,
+  and escalation to SIG TLs only if necessary.
+  - Each person who is an owner of any subproject of the SIG has one vote,
+    regardless of whether they are an owner of more than one subproject.
+- Seed members established at subproject founding
+- *MUST* be an escalation point for technical discussions and decisions in the subproject
+- *MUST* set milestone priorities or delegate this responsibility
+- *MUST* remain active in the role and are automatically removed from the position if they are unresponsive
```

Comment:
This seems like we should clarify unresponsive to exclude preplanned leave that the other sig leads are made aware of and some due dilligence to have responsibilities shouldered.  Ie maternity leave is not grounds for removal, bad faith radio silence is.

Reply:
This is copied verbatim from the [template](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance-template-short.md). Can we fix it there and then copy it out to all SIG charter drafts?

Reply:
I have language that reflects @smarterclayton sentiment in the sig-node charter I am writing with dawn.   I think it should be added in the global template and wanted to bring it up in steering committee for discussion in next meeting.

Reply:
FWIW, I have a longer template out that does a better job with this, but was considered too verbose by a number of of folks.  I'll work on getting the longer template merged and update the short template with something.

Link:
https://github.com/kubernetes/community/pull/1650/files#diff-16973d738c2229e3dac80e77e149200eR68

Reply:
Instead of copying it into each SIG charter, can we post a single copy somewhere and link to it from each SIG charter? Not for the whole template, but for things like these membership rules that we expect few SIGS to customize.